### PR TITLE
testcase : Add MediaPlayer utc

### PR DESCRIPTION
**Pull request by junmin-kim**
Add simple test of MediaPlayer methods.
getState, create, destroy, setDataSource,
prepare, unprepare, setVolume, getVolume

### CODE REVIEW

**Comment by junmin-kim** [comment]
@jsdosa @jarvisjang  Please review this PR

**Comment by seinfra** [comment]
Target : [1229b56589994561f7f2d35999b7ce5560e9be3b] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [1229b56589994561f7f2d35999b7ce5560e9be3b] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [0e1810850f1809e49814d2ce9e351f8363715a65] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [0e1810850f1809e49814d2ce9e351f8363715a65] - Code Rule Check OK.

File: apps/examples/testcase/ta_tc/media/utc/utc_media_player.cpp [hunk 1]
```diff
@@ -0,0 +1,100 @@
+#include <stdio.h>
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/media/utc/utc_media_player.cpp]: License

Comment (junmin-kim) [reply apps/examples/testcase/ta_tc/media/utc/utc_media_player.cpp]: Fixed it

**Comment by seinfra** [comment]
Target : [31426f23a55bb9378cf8fe4f90dc2462ff2f8216] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [0ca91f0d9aea612e26eba32006c450a473590e59] -

**Comment by seinfra** [comment]
Target : [0ca91f0d9aea612e26eba32006c450a473590e59] -

**Comment by seinfra** [comment]
Target : [9112195ba3cc6b6d4f6bdf8d92cda457e0484c35] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [9112195ba3cc6b6d4f6bdf8d92cda457e0484c35] - Code Rule Check OK.

**Pull request merged**